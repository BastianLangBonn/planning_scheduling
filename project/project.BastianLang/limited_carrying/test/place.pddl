(define (problem problem1)
	(:domain domain)
	(:objects 	s - s1
			o - o1
			r - robot
			c1 c2 - capacity)
	(:init 	(at_location r s)
		(grapped r o)	
		(robot_carries r c2)
		(preceding_capacities c1 c2)

		;Distance Metric

		(= (distance s s) 0)
		
		(= (total-cost) 0)
		
	)
	(:goal
		(and 
		(at_location o s)
		(not(grapped r o))
		(gripper_free r)
		(robot_carries r c1)
		(not(robot_carries r c2))
		)
	)
)
