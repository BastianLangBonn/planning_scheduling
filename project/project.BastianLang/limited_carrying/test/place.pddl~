(define (problem problem1)
	(:domain domain)
	(:objects 	s - s1
			o1 o2 o3 - o1)
	(:init 	(robot_at s)
		(staged o1)
		(staged o2)
		(staged o3)
		(carries_three)	

		;Distance Metric

		(= (distance s s) 0)
		
		(= (total-cost) 0)
		
	)
	(:goal
		(carries_none)
	)
)
