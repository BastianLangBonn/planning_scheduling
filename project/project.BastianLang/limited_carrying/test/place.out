(unstage o3)
(place o3 s)
(unstage o2)
(place o2 s)
(unstage o1)
(place o1 s)
